DevAcademy/Components/MapView/SymbolByPlaceKind.swift

```diff
@@ -1,17 +1,17 @@
 import SwiftUI
 
-struct PlaceMapAnnotationView: View {
+struct SymbolByPlaceKind: View {
```

Comment:
To puvodni pojmenovani `PlaceMapAnnotationView` mi prislo asi lepsi. Obecne u Views je dobre, aby ten nazev koncil slovem View (nebo treba Button, Field, apod.), aby bylo jasne, ze se jedna o UI prvek.
`SymbolByPlaceKind` na prvni dobrou zni jako kdyby se jednalo o nejaky Modelovy struct.